rules-emerging-threats/2023/Malware/Pikabot/proc_creation_win_malware_pikabot_rundll32_discovery.yml

```diff
@@ -0,0 +1,32 @@
+title: Suspicious Process Invocation by Rundll32 - Possibly Pikabot Discovery Activity
+id: 698d4431-514f-4c82-af4d-cf573872a9f5
+status: experimental
+description: |
+    Detects execution of rundll32 that leads to system discovery activity, such as incl. network, user info and domain groups.
+    The malware Pikabot have been seen to use this technique as part of its C2-botnet registration with a short collection timeframe (less than 1 minute).
+references:
+    - https://www.virustotal.com/gui/file/72f1a5476a845ea02344c9b7edecfe399f64b52409229edaf856fcb9535e3242
+author: Andreas Braathen (mnemonic.io)
+date: 2023/10/27
+tags:
+    - attack.discovery
+    - attack.t1016
+    - attack.t1049
+    - attack.t1087
+    - detection.emerging_threats
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection_grandparent:
+        GrandParentProcessName: 'rundll32.exe'
```

Comment:
GrandParent isn't an available field for most backends. So it can't be used here

Reply:
Supported on some backends such as based on MDE though, e.g. https://learn.microsoft.com/en-us/microsoft-365/security/defender/advanced-hunting-deviceprocessevents-table?view=o365-worldwide

Again, shouldn't the sigma-pipeline decide whether a valid mapping exists for the selected backend?